landoscript/docker.d/worker.yml

```diff
@@ -3,3 +3,4 @@ artifact_dir: { "$eval": "ARTIFACTS_DIR" }
 verbose: { "$eval": "VERBOSE == 'true'" }
 lando_api: { "$eval": "LANDO_API" }
 lando_token: { "$eval": "LANDO_TOKEN" }
+schema_file: /app/landoscript/src/landoscript/data/landoscript_task_schema.json
```

Comment:
Isn't this redundant with having it in the default config?

Reply:
uhh, yes!